collect_app/src/main/java/org/odk/collect/android/logic/FormController.java

```diff
@@ -1220,12 +1217,6 @@ public InstanceMetadata getSubmissionMetadata() {
             // timing element...
             v = e.getChildrenWithName(AUDIT);
             if (v.size() == 1) {
-                Collect.getInstance().getDefaultTracker()
```

Comment:
I would keep this since it's an evolving feature. It might also be good to log when location is turned on.